decision-making/voting-on-proposals.md

```diff
@@ -0,0 +1,31 @@
+# Voting on Proposals
+The Architecture team handles the proposal workflow and is responsible for procedural compliance.
+
+## Proposal submission
+- A proposal is submitted by opening a pull request to this repository.
+- The initiator must present the idea in an architecture meeting.
+- The announcement should include the problem the proposal is trying to resolve as well as any proposed solution(s).
+
+## Discussion and Voting
+- If there are controversial opinions in relation to the proposed idea, the architecture team may consider starting the voting.
+- The voting is considered open when the `voting` label is added to the pull request.
+- The voting can be represented as a single/multiple answer survey.
+- A valid voting period: 1) must be announced when the voting is started, 2) must be greater that 1 week and 3) must not be changed during the voting.
+- The Discussion may continue in the pull request thread.
+- Voters cannot change their vote during the voting period.
```

Comment:
Why not allow changing votes?

Reply:
My point was that it will lead to the carefully considered decision and more responsible attitude to the voting.

Reply:
I've had my mind and vote changed many times after some good discussions. I think the idea you are trying to address is valid but I think there may be some validity to changing it as well

Reply:
I'm not a psychologist, so maybe this is how people behave. So I'd just vote here to allow changing votes :) People may:
1. not notice that votes are not changeable until they really voted
1. click a wrong answer accidentally
1. vote and then see a discussion going in the PR (we still allow the discussion to go while voting is already in progress) and change their mind (thoughtfully)

Reply:
Makes sense. Updated this line.